test/advi_locscale.jl

```diff
@@ -0,0 +1,86 @@
+
+const PROGRESS = length(ARGS) > 0 && ARGS[1] == "--progress" ? true : false
+
+using Test
+
+@testset "advi" begin
+    @testset "locscale" begin
+        @testset "$(modelname) $(objname) $(realtype) $(adbackname)"  for
+            realtype ∈ [Float64], # Currently only tested against Float64
+            (modelname, modelconstr) ∈ Dict(
+                :NormalLogNormalMeanField => normallognormal_meanfield,
+            ),
+            (objname, objective) ∈ Dict(
+                :ADVIClosedFormEntropy  => ADVI(10),
+                :ADVIStickingTheLanding => ADVI(10, entropy = StickingTheLandingEntropy()),
+            ),
+            (adbackname, adbackend) ∈ Dict(
+                :ForwarDiff  => AutoForwardDiff(),
+                # :ReverseDiff => AutoReverseDiff(),
+                # :Zygote      => AutoZygote(), 
+                # :Enzyme      => AutoEnzyme(),
```

Comment:
What's the reason for only testing forward diff here?

Reply:
`DistributionsAD` doesn't work with `ReverseDiff`, as you know, `Bijectors.Stacked` is currently broken for `Zygote`, and `Enzyme` well.. doesn't work for our stuff yet. :(